task-impls/src/network.rs

```diff
@@ -190,7 +264,9 @@ impl<
             | SequencingHotShotEvent::ViewSyncVoteSend(_)
             | SequencingHotShotEvent::ViewSyncCertificateSend(_, _)
             | SequencingHotShotEvent::Shutdown
-            | SequencingHotShotEvent::ViewChange(_) => true,
+            | SequencingHotShotEvent::ViewChange(_)
+            | SequencingHotShotEvent::TransactionSend(_) => true,
+
```

Comment:
Do we still need this function, now that we've added different filters for the three types of networking tasks?